playground/frontend/playground_components/lib/src/theme/theme.dart

```diff
@@ -32,6 +32,9 @@ class BeamThemeExtension extends ThemeExtension<BeamThemeExtension> {
   final Color primaryBackgroundTextColor;
   final Color lightGreyBackgroundTextColor;
   final Color secondaryBackgroundColor;
+  // TODO(nausharipov): simplify new color addition
+  final Color selectedProgressColor;
+  final Color unselectedProgressColor;
```

Comment:
Can these use a single color with some transparency?

Reply:
The selected progress indicator is darker in light mode and lighter in dark mode compared to the unselected state. AFAIK, a semi-transparent indicator would look differently.